README.md

```diff
@@ -484,6 +484,7 @@ After setting this and issuing a `jira login`, your credentials will be stored i
 An alternative to the keyring password source is the `pass` tool (documentation [here](https://www.passwordstore.org/)).  This uses gpg to encrypt/decrypt passwords on demand and by using `gpg-agent` you can cache the gpg credentials for a period of time so you will not be prompted repeatedly for decrypting the passwords.  The advantage over the keyring integration is that `pass` can be used on more platforms than OSX and Linux, although it does require more setup.  To use `pass` for password storage and retrieval via `go-jira` just add this configuration to `$HOME/.jira.d/config.yml`:
 ```yaml
 password-source: pass
+passwrod-name: jira.example.com/myuser
```

Comment:
There is a typo: `passwrod` instead of `password`.

Reply:
Fixed the typo.